static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -315,6 +312,41 @@ export function DisasterEventMapTile(
       });
   }
 
+  /**
+   * Updates breadcrumbs using URL params and current place.
+   */
+  function updateBreadcrumbs(selectedPlace: NamedTypedPlace): void {
```

Comment:
we could technically compute this from parent places. add a TODO, so we won't rely on a URL param?

Reply:
added a TODO so we can revisit this, but I didn't want to rely on parent places because there may be parent places that weren't seen by the user. 

E.g., parent places for USA are: Earth, North America. However, if the user clicked on USA from the Earth page, we wouldn't want North America to show up in the breadcrumbs.

Reply:
i've always seen breadcrumbs as a way to learn the navigation (so if USA was the first page you clicked on, you would learn that there is also an Earth page).  just like how we have it on docsite. less so of a navigation history tool.

Reply:
I see, if it's more of a navigation tool, then using parent places makes sense. Although, we might want to hide continents